webapp/app/components/common/modal/MsModal.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import PropTypes from 'prop-types';
+
+class MsModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      modal: false
+    };
+
+    this.toggle = this.toggle.bind(this);
+  }
+
+  toggle() {
+    this.setState({
+      modal: !this.state.modal
+    });
+  }
+
+  render() {
+    return (
+      <div>
+        <Modal isOpen={this.props.isOpen} toggle={this.props.toggle}>
+          <ModalHeader toggle={this.props.toggle}>{this.props.modalTitle}</ModalHeader>
+          <ModalBody>
+            {this.props.body}
+          </ModalBody>
+          <ModalFooter>
+            <Button color="primary" onClick={this.props.toggle}>{this.props.okButtonLabel}</Button>{' '}
```

Comment:
I would avoid computed whitespace as a general rule, preffer to style and add desired spacing via Stylesheet, or inline styles either adding padding to the buttons or the desired margin

Reply:
Done, that remained from before your previous comment.